guides/support-and-on-call.md

```diff
@@ -146,9 +149,9 @@ Outside office hours, the expected response depends on the severity of the
 incident. We have a set of factors that determine the severity of an incident,
 and have included these in our [Service Level Agreement].
 
-[GovPress hosting]: https://www.digitalmarketplace.service.gov.uk/g-cloud/services/930612236449495
+[govpress hosting]: https://www.digitalmarketplace.service.gov.uk/g-cloud/services/930612236449495
```

Comment:
It's a bit confusing having these in a different capitalisation than in the body text. I guess it's good that the markdown parser? interpreter? recognises them anyway, but is it syntactically incorrect to leave them capitalised the same way as in the body?